Sources/StreamChat/Controllers/ConnectionController/ConnectionController.swift

```diff
@@ -0,0 +1,254 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+import Foundation
+
+public extension _ChatClient {
+    /// Creates a new `ChatConnectionController` instance.
+    ///
+    /// - Returns: A new instance of `ChatConnectionController`.
+    ///
+    func connectionController() -> _ChatConnectionController<ExtraData> {
+        .init(client: self)
+    }
+}
+
+/// `ChatConnectionController` is a controller class which allows to explicitly
+/// connect/disconnect the `ChatClient` and observe connection events.
+///
+/// Learn more about `ChatConnectionController` and its usage in our [cheat sheet](https://github.com/GetStream/stream-chat-swift/wiki/StreamChat-SDK-Cheat-Sheet#connection).
```

Comment:
This link doesn't work :) Can you please fix it and also create the section in the cheatsheet?

Reply:
I followed the same approach of the current user controller where it pointed to a #user, but it also does not exist for the CurrentUserController. Should both point to the Controllers-Overview#CurrentUserController and Controllers-Overview#ChatConnectionController instead? Or the goal is to create a page per controller?

Reply:
Will fix it in the CurrentUserController, since the link is wrong there too 👍